GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -755,31 +755,42 @@ private void RegisterPlugins()
 
             lock (PluginRegistry.Plugins)
             {
-                var pluginEntries = PluginRegistry.Plugins
+                var pluginManager = PluginRegistry.Plugins.FirstOrDefault(p => p.Name == "Plugin Manager");
```

Comment:
This isn't a localisable property, is it?

Reply:
> This isn't a localisable property, is it?

Valid question.

Reply:
I looked at that before doing the development and from what I see, no, that's not localisable.

`<joke>But nothing is localized in plugins 🤣 </joke>`

In fact, `Name` is used as the key to store the settings, so I hope it won't be localisable anytime soon....